programs/programs/rental_assistance_grant/calculator.py

```diff
@@ -0,0 +1,48 @@
+from programs.programs.calc import ProgramCalculator, Eligibility
+import programs.programs.messages as messages
+from programs.co_county_zips import counties_from_zip
+from programs.sheets import sheets_get_data
+from integrations.util import Cache
+
+class RentalAssistanceGrant(ProgramCalculator):
+    amount = 10_000
+
+    def eligible(self) -> Eligibility:
+        e = Eligibility()
+
+        # location
+        counties = counties_from_zip(self.screen.zipcode)
+        county_name = counties[0] if len(counties) > 0 else self.screen.county
```

Comment:
We want the zipcode to be a backup if the screen does not have a county.
```suggestion
        county_name = self.screen.county if self.screen.county is not None else counties[0]
```